suricata.yaml.in

```diff
@@ -1470,6 +1470,13 @@ flow:
   #managers: 1 # default to one flow manager
   #recyclers: 1 # default to one flow recycler thread
 
+# Mark a flow as an elephant flow if the rate of influx of bytes is equal to
+# or exceeds the given number. The rate is calculated by dividing the total
+# number of bytes a flow has seen in either direction divided by the time a
+# flow has been alive.
+elephant-flow:
+  rate: 1000000
```

Comment:
to me a rate needs a time element, how would that be expressed here?

Reply:
yeah. will make it take two args instead: bytes and seconds just like the `flow.rate` keyword